test/overall/compute.mocha.js

```diff
@@ -381,6 +381,56 @@ describe("compute native", () => {
       .done();
   });
 
+  it("works with quantiles", (testComplete) => {
+    // Test data comes from: https://en.wikipedia.org/wiki/Quantile
+    var quantileData = [
+      { vOdd:  3, vEven:  3 },
+      { vOdd:  6, vEven:  6 },
+      { vOdd:  7, vEven:  7 },
+      { vOdd:  8, vEven:  8 },
+      { vOdd:  8, vEven:  8 },
+      { vOdd:  9, vEven: null },
+      { vOdd: 10, vEven: 10 },
+      { vOdd: 13, vEven: 13 },
+      { vOdd: 15, vEven: 15 },
+      { vOdd: 16, vEven: 16 },
+      { vOdd: 20, vEven: 20 }
```

Comment:
Nope it is not required, the data is treated as a set. Will update the docs and the test case.